Assets/GothicVR/Scripts/World/GameTime.cs

```diff
@@ -21,16 +26,25 @@ public class GameTime : SingletonBehaviour<GameTime>
         private static readonly float ONE_INGAME_SECOND = 0.06944f;
         private DateTime time = new(1, 1, 1, 15, 0, 0);
 
-        public GameObject sun;
+        private void OnEnable()
+        {
+            //Subscribe to Events
+            secondChangeChannel.OnEventRaised += RaiseMinuteEvent;
```

Comment:
Please think about having this easier. It took me a while to understand why the secondEvent raises the MinuteEvent etc.

Why not having one method which is called every ingame second (you have it already) and then check there inside if a minute or hour is to be raised.

If there's value of this abstraction I don't understand: Cool. If not, let's think about the more direct approach as mentioned above.

Reply:
Done